cmd/telegraf/telegraf.go

```diff
@@ -267,6 +275,18 @@ func main() {
 		processorFilters = strings.Split(":"+strings.TrimSpace(*fProcessorFilters)+":", ":")
 	}
 
+	if *pprofAddr != "" {
+		go func() {
+			log.Printf(
+				"I! Starting pprof on %s. Open profiling tools page in browser: /debug/pprof",
```

Comment:
@danielnelson, thanks for review. Changes done.

Could you also move forward our contributions: 
https://github.com/influxdata/telegraf/pull/2387
https://github.com/influxdata/telegraf/pull/2594

We hope to see them in 1.3 too